### ISSUE TITLE
Fix the Load More functionality

### ISSUE THREAD

**Issue by s-Hale** [s-Hale/BuildTravelCards#73]
Function errors out when you hit load more at the end of the selection. Can this be fixed?

**Comment by s-Hale** [comment]
<img width="1062" alt="Screenshot 2019-10-09 at 11 32 14" src="https://user-images.githubusercontent.com/18394337/66474167-a834df80-ea88-11e9-9c92-b911f4aad230.png">

**Comment by mhmoosazadeh** [comment]
Hi @s-Hale, Can I fix it?

**Comment by mhmoosazadeh** [comment]
@s-Hale Could you tell me when this error occurred?

**Comment by s-Hale** [comment]
No, to be honest, I merged in a couple of additional journeys but I'm at work and can't look into this more thoroughly.
